### DISCUSSION TITLE
Should we keep `.environment` file?

### DISCUSSION THREAD

**Discussion by lolautruche** [platformsh/platformify#64] [Ideas]

**Comment by akalipetis** [comment]
@lolautruche I converted this to a discussion, as it's not immediately actionable. I believe we should keep this, as it helps us bridge the gap and be able to easily migrate projects that are used to get variables from the environment.

Even if they use different variables, it's always easier to change a name in there and be ready than to search for that IMO.

What's the counter argument?
